.bazelrc

```diff
@@ -33,16 +33,6 @@ build --incompatible_merge_fixed_and_default_shell_env
 # modern python versions (3.9+)
 build --incompatible_default_to_explicit_init_py
 
-###############################################################################
-## Bzlmod
-###############################################################################
-
-# TODO: migrate all dependencies from WORKSPACE to MODULE.bazel
-common --noenable_bzlmod
-
-# Disable the bzlmod lockfile, so we don't accidentally commit MODULE.bazel.lock
-common --lockfile_mode=off
```

Comment:
Isn't enabling this just gonna add a volatile file to the repo?

Reply:
It is not volatile, you're supposed to commit MODULE.bazel.lock.  For us it should include almost nothing.